src/widgets/dialogs/UserInfoPopup.cpp

```diff
@@ -945,44 +945,49 @@ void UserInfoPopup::updateUserData()
         this->ui_.block->setEnabled(true);
         this->ui_.ignoreHighlights->setChecked(isIgnoringHighlights);
 
-        // get followage and subage
-        getIvr()->getSubage(
-            this->userName_, this->underlyingChannel_->getName(),
-            [this, hack](const IvrSubage &subageInfo) {
-                if (!hack.lock())
-                {
-                    return;
-                }
+        auto type = this->channel_->getType();
```

Comment:
```suggestion
        auto type = this->underlyingChannel_->getType();
```

`channel_` and `underlyingChannel_` can be different. I think you could check for `type == Channel::Type::Twitch`. Any other channel will have a bad name like `/live`, `/watching`, `/automod`, etc. (note: this doesn't prevent loading in usercards from these channels completely - they will set the channels appropriately where possible). Just in the live channel, there's no reasonable channel to set.